doc/multiBand.dox

```diff
@@ -0,0 +1,73 @@
+namespace lsst { namespace pipe { namespace tasks {
+/**
+\page pipeTasks_multiBand High-level Overview of Multi-Band Coadd Processing
+
+The goal of coadd measurement is to consistently detect and measure sources across coadds constructed in 
+multiple filter bands in a consistent and reasonable manner. Specifically, we would like to measure the same 
+(deblended) sources across all photometric bands. There are several advantages to imposing this requirement. 
+We get the same deblending in all the bands. This allows us to run forced photometry on consistently deblended 
+objects which gives us reasonable colors. It also permits us to obtain measurements of flux in the case of 
+drop-out objects such as Lyman-break galaxies. Our strategy for meeting the consistent deblending requirement 
+can be broken up into a number of simple steps.
+<DL> 
+  <DT> i. \ref DetectCoaddSourcesTask_ "DetectCoaddSourcesTask": Detect Sources in Individual Coadds</DT> 
```

Comment:
Some of the tasks mentioned here seem to get linked to their documentation, but not all of them. I'm not sure if that's just an artefact of the way I built the documentation, or if it's because not all of the tasks are actually documented, or something else.